### PR TITLE
PWA support added

### PR DESCRIPTION
**Pull request by officialpm**
# PWA Support for Python's Course LMS

 - [x] favicon.ico path fix 🔧
 - [x] Added Icons 🎨
 - [x] Added Manifest.json 🌐
 - [x] Added service worker (sw.js) 👷
 - [x] Added routes - /manifest.json, /sw.js ➕

#### Solves issue #164 🤘🏻

### CODE REVIEW

File: lms/templates/base.html [hunk 1]
```diff
@@ -9,6 +9,25 @@
     <link rel="stylesheet" href="{{ url_for('static', filename='my.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='prism.css') }}">
     <link rel="stylesheet" href="{{ url_for('static', filename='hawcons.css') }}">
+    <link rel="manifest" href="{{ url_for('static', filename='manifest.json') }}">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="transparent">
+    <link rel="apple-touch-icon" sizes="57x57" href="{{ url_for('static', filename = 'icons/apple-icon-57x57.png')}}">
```
Comment (yammesicka) [inline lms/templates/base.html:15]: This is a very large payload. We should allow only 180x180. See [this](https://stackoverflow.com/a/21144916/1058671) StackOverflow answer.

**Review by yammesicka — COMMENTED** [review]
Thank you for your contribution, you are awesome :+1: 

Please address the required changes :)

File: lms/static/manifest.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Python's Course LMS",
+  "short_name": "LMS",
+  "theme_color": "#1290f4",
+  "background_color": "#2196f3",
+  "display": "standalone",
+  "scope": "https://a466824bc7ed.ngrok.io/login?next=%2F",
+  "start_url": "https://a466824bc7ed.ngrok.io/login?next=%2F",
```
[the remark this answers is not in the document]
Comment (officialpm) [reply lms/static/manifest.json:8]: Oh sorry, committed the local config!

Yes we can generate it dynamically🤘🏻 
Can u please help me with config path?

File: lms/static/sw.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+console.log('Hello from sw.js');
+
+importScripts('https://storage.googleapis.com/workbox-cdn/releases/3.2.0/workbox-sw.js');
```
[the remark this answers is not in the document]
Comment (officialpm) [reply lms/static/sw.js:3]: Yes will do it!

File: lms/templates/base.html (hunk 1 above)
Comment (officialpm) [reply lms/templates/base.html:15]: Ok will look into it!

File: lms/static/manifest.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (yammesicka) [reply lms/static/manifest.json:8]: Sure :)
The config is currently under lms/lmsweb/config.py.example

**Pull request closed without merging**